lib/react.js

```diff
@@ -134,6 +134,7 @@ declare type React$AbstractComponentStatics = {
   // This is only on function components, but trying to access name when
   // displayName is undefined is a common pattern.
   name?: ?string,
+  propTypes?: any,
```

Comment:
@jbrown215 This will not let to enable dynamic-export lint ever

Reply:
Thanks for bringing that up. I am discussing with @dsainati1 and we'll figure out a way to move forward.

Reply:
Thank you for following up on this :)